.travis.yml

```diff
@@ -49,6 +49,7 @@ script:
     }
   - buildExpr=$(getBuildExpr)
   - time nix-instantiate -E "$buildExpr" --add-root ./drv --indirect
+  - nix show-derivation ./drv
```

Comment:
I think this is too noisy, e.g. look at this build output: https://travis-ci.org/fort-nix/nix-bitcoin/jobs/654613360
What was your reason to add this?

Reply:
Hm, that's indeed too noisy. Reason was to figure out how my local derivation differed from what is built on travis (which turned out to be the wrong `numCPUs` and `memoryMiB` as mentioned above). IIRC if you cat on travis it doesn't print the output unless you expand that section.

Reply:
Hmm perhaps that only happens in `after_script` and not `script`.

Reply:
Yeah, Travis doesn't show the `cat` output.
`nproc` and memory are the only build host variables that influence the test derivation.
I've added https://github.com/fort-nix/nix-bitcoin/pull/140/files to show these in full detail.
I think that's good enough.